roles/openshift_efk/README.md

```diff
@@ -0,0 +1,9 @@
+###Required vars:
+
+- kibana_hostname: kibana.example.com
+- es_cluster_size: 1
+- master_url: https://localhost:8443
```

Comment:
The additional benefit to this naming scheme is that we can more easily add the ability to persist these facts with openshift_facts down the line.